src/Strategy/Bundler.hs

```diff
@@ -132,7 +136,10 @@ analyzeGemfileLock :: (Has ReadFS sig m, Has Diagnostics sig m) => BundlerProjec
 analyzeGemfileLock project =
   warnOnErr AllDirectDeps
     . warnOnErr MissingEdges
-    . errCtx (BundlerMissingLockFile $ bundlerGemfile project)
+    . errCtx (BundlerMissingLockFileCtx $ bundlerGemfile project)
```

Comment:
I think this is a widespread problem, but I encountered it while testing this, so I'll leave the comment here.

I created a ruby project and deleted the `Gemfile.lock` file. This creates a warning in the scan summary.

On master, the scan summary does not contain colour codes. On this branch it does.

The colour codes are helpful if you `cat` the scan-summary, but if you open it in an editor they don't get rendered properly. Here's what it looks like in VS Code:

<img width="1560" alt="CleanShot 2024-02-02 at 16 55 07@2x" src="https://github.com/fossas/fossa-cli/assets/13045/1a43cc20-ffef-4f81-875d-2471ad209f79">

Do we want to remove the colour codes when generating scan-summary.txt?

Reply:
This isn't a problem with the colour codes, it's just that VSCode doesn't display color codes by default. 

Installing the ANSI colors extension gives you a nice view of the file: https://marketplace.visualstudio.com/items?itemName=iliazeus.vscode-ansi

<img width="1511" alt="CleanShot 2024-02-02 at 17 01 25@2x" src="https://github.com/fossas/fossa-cli/assets/13045/47c96155-edcc-4776-bbca-50e2f3417857">

Reply:
After playing around with this, in order to remove the ANSI color codes from the doc we need to do some regex to parse the doc as a string and remove the color codes that way. 

I'm wondering if we should just keep it as is and have something in the `HACKING.md` to direct ppl to download the extension you listed so that we can view the colors in VSCode

Reply:
Unfortunately I don't think putting a comment in HACKING.md will help much. The scan-summary is used by users, and the audience for HACKING.md is people who are submitting patches to the CLI.

Let's ask in the #team-analysis channel about this. I really don't know if users look at this file very often, but I think they might and this might be worth fixing.

There is a package to do this, but I haven't looked into it other than seeing that it exists: https://hackage.haskell.org/package/strip-ansi-escape

Reply:
Updated to remove the ANSI color codes form ScanSummary 

![Screenshot 2024-02-09 at 4 36 15 PM](https://github.com/fossas/fossa-cli/assets/28590628/ef76be90-0131-4ccf-95d9-1f49a00f5629)